### ISSUE TITLE
Review annotations to protein dimerization activity (GO:0046983)

### ISSUE THREAD

**Issue by vanaukenk** [geneontology/go-annotation#2586]
To continue the work on revising annotations made to GO terms that reflect stoichiometry of interactions, we would like curators to review a subset of manual annotations to 'protein dimerization activity' (GO:0046983).

A few things for curators to note:

1. In the spreadsheet, I've listed all existing annotations, but only those with a colored background need to be manually reviewed.  The other annotations should be able to be changed en masse programmatically (depending on how groups handle the bulk changes).

2. As the suggested changes to existing annotations depends on the evidence code used, I'm going to put annotations for each evidence code in a separate comment to hopefully make it easier for curators to hone in on the ones they need to review.

Any questions, please add a comment to the ticket.  Thx.

https://docs.google.com/spreadsheets/d/1ymWzjGnFhkJ9oZTX-bX_nENkKmASQW1dM3eISOa72dg

**Comment by vanaukenk** [comment]
**IDA**

Annotations to GO:00046983 (protein dimerization activity) and evidence code ECO:0000314 (IDA)
		Issue: no entry in the With/From field
		Need manual review to see if they can be changed (see spreadsheet)
		How many of these and who?

			**AgBase - 2
			ARUK-UCL - 1
			BHF-UCL - 2
			CAFA - 1
			dictyBase - 2
			FlyBase - 2
			GeneDB - 8
			MGI - 7 (fixed)
			RGD - 11 (fixed)
			UniProt - 41
			ZFIN - 3**

**Comment by vanaukenk** [comment]
**NAS**

Annotations to GO:00046983 (protein dimerization activity) and evidence code ECO:0000303 (NAS)
		Issue: no entry in the With/From field
		Need manual review to see if they can be changed to an experimental code
		If not, delete the annotation
		How many of these and who?
			**AgBase - 1
			FlyBase - 1
			UniProt - 6**

**Comment by vanaukenk** [comment]
**IMP**

Annotation to Annotations to GO:00046983 (protein dimerization activity) and evidence code ECO:0000315 (IMP)
		Issue: no entry in the With/From field
		Need manual review to see if they can be changed to IPI (if not already there)
			**AgBase - 1
			MGI - 2
			RGD -2 (fixed)**

**Comment by vanaukenk** [comment]
**RCA**

Annotations to GO:00046983 (protein dimerization activity) and evidence code ECO:0000245 (RCA)
		Issue: no entry in the With/From field
		Need manual review to see if they can be changed to an experimental code
		If not, delete the annotation
		How many of these and who?
			**AgBase - 4
			GR - 17**

**Comment by vanaukenk** [comment]
**TAS**

Annotations to GO:00046983 (protein dimerization activity) and evidence code ECO:0000304 (TAS)
		Issue: no entry in the With/From field
		Need manual review to see if they can be changed to an experimental code
		If not, delete the annotation
		How many of these and who?
			**DFLAT - 1
			GeneDB - 4
			MGI - 2 (fixed)
			RGD - 3 (fixed)
			UniProt - 3**

**Comment by jaiswalp** [comment]
please delete these annotations GR - 17

**Comment by hattrill** [comment]
FB done

**Comment by sabrinatoro** [comment]
ZFIN annotations made with IDA have been changed. 

@pgaudet @vanaukenk : Just to make sure: I should ask the ZFIN developers to update the other "protein dimerization activity" annotations to "protein binding" (with IPI). Is that correct (this is happening at the Mod)?
Is there a ticket where this work should be reported when done? 
Thanks!

**Comment by vanaukenk** [comment]
@sabrinatoro 

Yes, the ZFIN developers can update 'protein dimerization activity' annotations that use IPI to 'protein binding'.
The full set of recommendations for this terms is in this Google doc (in the annotation review folder):
https://docs.google.com/document/d/1JxzNwl1fr5Fj-IJy_Fo1lLfAlAQB8IxPDwwYKxIXBZo

Let us know if you have any other questions.

**Comment by RLovering** [comment]
UCL done

**Comment by pgarmiri** [comment]
UniProt-EBI annotations have been checked and actions recorded in the spreadsheet. SWIS and SWIP annotations are tagged.

**Comment by pgaudet** [comment]
Some annotations do not have 'with' 


Group | Count 
-- | --
UniProt | 28
RGD | 1
CAFA | 1
MGI | 2
dictyBase | 1
GeneDB | 4
AgBase | 3

(lines 70-104 in the new spreadsheet here: https://docs.google.com/spreadsheets/d/1Nw_ppWc85CiHayvxgjvqwRoPSIAACfEUPGfwZslANkg/edit?gid=0#gid=0)
